armi/materials/tests/test_materials.py

```diff
@@ -80,6 +80,10 @@ def test_duplicate(self):
         .. test:: Test the material base class.
             :id: T_ARMI_MAT_PROPERTIES3
             :tests: R_ARMI_MAT_PROPERTIES
+
+        .. test:: Materials shall calc mass fracs at init.
+            :id: T_ARMI_MAT_FRACS
+            :tests: R_ARMI_MAT_FRACS
```

Comment:
this test is on a protected test class and doesn't show up in the API.

Reply:
same with the other test tags here. Either they all need to be moved or the class needs to be made public.

Reply:
Dammit

Reply:
Drat

Reply:
Okay, this private test class had a BUNCH of req crumbs. I moved them all to a public test class on a specific material.